core/graph/README.md

```diff
@@ -54,4 +54,104 @@ to be run before and after an operation chain is executed.
 
 You can use hooks to do things like custom logging or special operation
 chain authorisation. To implement your own hook, just implement the `GraphHook`
-interface and register it with the graph when you build a `Graph` instance.
\ No newline at end of file
+interface and register it with the graph when you build a `Graph` instance.
+
+## FAQs
+To use an example of the authorised Graph exporter within the road-traffic-demo using the proxy-store, follow these steps.

```

Comment:
Please could you add a sub heading, similar to the FAQs in core/operation/README.md.


## FAQs
Here are some frequently asked questions.

#### How can I run an example of the authorised Graph exporter?
To use an example of the authorised Graph exporter within the road-traffic-demo using the proxy-store, follow these steps.

etc.